start-server.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
-echo "Starting test server"
+echo "Starting test server with DEBUG set to False"
```

Comment:
Is this always and only for debug? 

Maybe switch the order and have `echo "Starting test server with DEBUG set to ${DEBUG}"` ?

Reply:
actually I meant to remove that. Thanks.